includes/diagnostics/class-wc-stripe-diagnostics-trace-store.php

```
@@ -0,0 +1,595 @@
+					return false;
+				}
+				$index = $this->read_index();
+				if ( ! in_array( $session_id, $index, true ) ) {
+					$index[] = $session_id;
+				}
+				$index = $this->evict_oldest( $index );
+				$this->write_index( $index );
+				return true;
+			}
+		);
+	}
```

PHPMD (2.15.0)

[error] The parameter $session_id is not named in camelCase. (undefined)

(CamelCaseParameterName)